### PR TITLE
update upload.sh to upload mysql-metadata-storage tarball too

### PR DESCRIPTION
**Pull request by himanshug**

### CODE REVIEW

File: upload.sh [hunk 1]
```diff
@@ -10,12 +10,19 @@ if [ $# -lt 1 ]; then
...
   exit 2
 fi
 
-s3cmd put distribution/target/$TAR $S3PATH/$TAR
+if [ ! -z "`s3cmd ls "$S3PATH/$MYSQL_TAR"`" ]; then
+  echo "ERROR: Refusing to overwrite $S3PATH/$MYSQL_TAR" >&2
+  exit 2
+fi
+
+echo s3cmd put distribution/target/$DRUID_TAR $S3PATH/$DRUID_TAR
```
Comment (xvrl) [inline upload.sh]: why prefix with echo?

Comment (himanshug) [inline upload.sh]: duh ... its a mistake, i was testing it and left the echo in place.

Comment (himanshug) [inline upload.sh]: fixed

**Comment by xvrl** [comment]
:+1:

**Comment by nishantmonu51** [comment]
:+1:

**Pull request merged**